### DISCUSSION TITLE
New Storage Request

### DISCUSSION THREAD

**Discussion by beatriz-mano** [biigle/core#775] [Q&A]
Hi, I already converted all my TIF files into PNG files and I tried to request the new storage and paper this popup 'The server didn't respond, sorry.' Can you help me?

**Comment by dlangenk** [comment]
Hi Beatriz,

This sometimes happens when uploading takes too long, or the pc goes into hibernation or energy saving mode. If you still have the window open just click the submit button again. Otherwise, please try uploading it again. I'll have a look at the storage-requests today.

Best
Daniel
